platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -165,9 +165,9 @@ public static class ClassVersion {
 
     /**
      * The number of seconds to wait for the full leaf rehash process to finish
-     * (see {@link VirtualRootNode#fullLeafRehash()}) before we fail with an exception.
+     * (see {@link VirtualRootNode#fullLeafRehashIfNecessary()}) before we fail with an exception.
      */
-    private static final int MAX_FULL_REHASHING_TIMEOUT = 21600; // 6 hours
+    private static final int MAX_FULL_REHASHING_TIMEOUT = 3600; // 1 hour
```

Comment:
Why is it no longer needed to be 6h?

Reply:
When I added this code, I didn't know that we have 5 minutes SLA for the upgrade. I left 1 hour as a theoretical limit for the testing purposes.